ftw/noticeboard/profiles/example_workflow/workflows.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="portal_workflow" meta_type="Plone Workflow Tool">
+    <object name="noticeboard_notice_workflow" meta_type="Workflow"/>
```

Comment:
In diesem Pull Request geht es um Category, nicht um Notice, oder? 
noticeboard_notice_workflow -> noticeboard_category_workflow